src/main/configurations/Translate/CreeerZaak_LK01/xsl/CreateZgwAndereZaakBody.xsl

```diff
@@ -1,11 +1,36 @@
-<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="2.0">
-	<xsl:output method="xml" version="1.0" encoding="UTF-8" indent="yes"/>
-    <xsl:param name="Url"/>
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns:xs="http://www.w3.org/2001/XMLSchema" version="2.0">
+    <xsl:output method="xml" version="1.0" encoding="UTF-8" indent="yes" />
+    <xsl:param name="VerwerkingsSoort"/>
+    <xsl:param name="AndereUrl"/>
+    <xsl:param name="AndereUrls" select="parse-xml('&lt;urls&gt;&lt;url/&gt;&lt;/urls&gt;')" />
```

Comment:
Specifying a default for "node()" types you can do like this:
`<xsl:param name="AndereUrls" as="node()"><urls><url/></urls></xsl:param>`

That is a bit more readable.